src/Aspire.Hosting/ApplicationModel/ResourceAnnotationMutationBehavior.cs

```diff
@@ -3,8 +3,17 @@
 
 namespace Aspire.Hosting.ApplicationModel;
 
+/// <summary>
+/// TODO: Doc Comments
```

Comment:
Don't know if you did already, but it'd be great if we also log an issue for all of these TODO's that were added. Of course just a single issue for all is sufficient.

Reply:
I plan on logging it once this change is merged.

Reply:
Logged https://github.com/dotnet/aspire/issues/2213.